examples/css/index.html

```diff
@@ -82,7 +82,27 @@ <h3 class="mistica-text-title2">Tag</h3>
                     <div class="mistica-tag-warning">Warning</div>
                     <div class="mistica-tag-error">Error</div>
                 </div>
-
+                <h3 class="mistica-text-title2">Counter</h3>
+                <div class="mistica-counter">
+                    <button class="mistica-counter__button" aria-label="decrease" disabled>
```

Comment:
in AEM, we should allow configuring aria-label for the input and increase/decrease buttons